src/transformers/tokenization_squeezebert.py

```diff
@@ -0,0 +1,77 @@
+# coding=utf-8
+#
```

Comment:
Should update the copyright, `The SqueezeBert authors and The HuggingFace Inc. team.`

Reply:
Done.